src/views/forms/TransferForm.tsx

```diff
@@ -505,3 +484,106 @@ const $FormInputToggleButton = styled(ToggleButton)`
 
   --button-padding: 0 0.5rem;
 `;
+
+function useForm(initialToUsdc: boolean) {
```

Comment:
maybe move to new file `transferHooks` to keep consistent with the Vault file structure? I think it would require putting the TransferForm into a TransferForm folder as well